Solutions/CohesitySecurity/cohesity.json

```diff
@@ -1,22 +1,28 @@
 {
     "api_key": "<cohesity-helios-api-key>",
     "resource_group": "<resource-group-name>",
+    "resource_type": "Microsoft.OperationalInsights/workspaces",
     "automation_account_name": "MyAutomationAccount",
     "producer_fun_prefix": "cohesitypro",
     "producer_context": "IncidentProducer",
     "consumer_fun_prefix": "cohesitycon",
     "consumer_context": "IncidentConsumer",
     "subscription_id": "********",
     "client_id": "********",
+    "object_id": "********",
     "client_secret": "********",
     "client_key": "********",
     "start_days_ago": "-30",
+    "location": "********",
     "user_email": "********",
     "resource_url": "https://management.azure.com/",
     "azure_url_scope": "https://management.azure.com/.default",
     "workspace_id": "********",
     "alert_id": "5461471069190603:1677650940946971",
     "tenant_id": "********",
     "blob_container_name": "cohesity-extra-parameters",
-    "workspace_name": "<workspace-name>"
+    "workspace_name": "<workspace-name>",
+    "service_now_username": "********",
```

Comment:
I have not seen any of the service_now parameters used in this PR. Please add them in a future PR when it's used.

Reply:
they are used in test.

Reply:
Solutions/CohesitySecurity/Tests/az.test.py:46             self.service_now_username = config["service_now_username"]